lib/containers/utils.pm

```diff
@@ -96,30 +96,23 @@ sub test_update_cmd {
         return;
     }
 
-    # The default values for these options are 0.
-    # Before adding more options check that they're supported by both podman & docker
-    my @opts = ('cpu-shares', 'memory-swap');
+    my $old_value = script_output "$runtime container inspect -f '{{.HostConfig.CpuShares}}' $container";
+    die "Default for cpu-shares != 0" if ($old_value != 0);
 
-    foreach my $opt (@opts) {
-        # Transform 'cpu-shares' into 'CpuShares'
-        (my $param = $opt) =~ s/^(.)(.*)-(.)(.*)/\u$1$2\u$3$4/;
+    my $try_value = 512;
 
-        my $old_value = script_output "$runtime container inspect -f '{{.HostConfig.$param}}' $container";
-        die "Default for $opt != 0" if ($old_value != 0);
+    assert_script_run "$runtime update --cpu-shares $try_value $container";
```

Comment:
I guess this this is a sync operation? I mean the command ends when the container effectively has update